src/components/ReplyComment/index.js

```diff
@@ -95,7 +95,7 @@ const ReplyCommentId = ({
       <ScrollView ref={scrollViewRef} contentContainerStyle={styles.commentScrollView}>
         <View style={styles.containerComment}>
           <ReplyCommentItem
-            feedId={dataFeed.id}
+            feedId={dataFeed?.id}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `feedId={dataFeed.id}` to `feedId={dataFeed?.id}` is a good one. It ensures that if `dataFeed` is null or undefined, the code won't throw an error trying to access the `id` property. This improves the robustness of the code.
```diff
- feedId={dataFeed.id}
+ feedId={dataFeed?.id}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->